cassava.cabal

```diff
@@ -67,6 +72,9 @@ Library
     unordered-containers < 0.3,
     vector < 0.12
 
+  if impl(ghc >= 7.4 && < 7.6)
+    build-depends: ghc-prim
```

Comment:
maybe just a short single-line comment that this is for GHC.Generics would be nice